containerbuddy/memcached.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
this does need some attention, I think prefixing the event would be a good idea.